src/argus_ticket_rt.py

```diff
@@ -149,11 +161,16 @@ def create_ticket(cls, serialized_incident: dict):
             LOG.exception(queue_error)
             raise TicketSettingsException(queue_error)
 
-        body = cls.create_html_body(serialized_incident=serialized_incident)
-        custom_fields = cls.get_custom_fields(
+        custom_fields, missing_fields = cls.get_custom_fields(
             ticket_information=ticket_information,
             serialized_incident=serialized_incident,
         )
+        body = cls.create_html_body(
+            serialized_incident={
+                **serialized_incident,
+                **{"missing_fields": missing_fields},
+            }
+        )
```

Comment:
I don't understand why you would create the dict this way.

You could just do

```
body = cls.create_html_body(serialized_incident={missing_fields: missing_fields, **serialized_incident})
```

or even

```
context = serialized_incident
context["missing_fields"] = missing_fields
body = cls.create_html_body(serialized_incident=context)
```

I would though recommend that "missing_fields" is renamed to something that won't already be on an incident though: "__missing_fields__" maybe.

Reply:
WAT, double underscore is interpreted as *bold*. Markdown, how I hate thee.